backend/db/ddl-11-all_csets.jinja.sql

```diff
@@ -55,6 +55,6 @@ SELECT ac.*, cscnt.counts, cscnt.counts->>'Members' as concepts
 FROM ac
 LEFT JOIN {{schema}}codeset_counts cscnt ON ac.codeset_id = cscnt.codeset_id;
 
-CREATE INDEX ac_idx1 ON {{schema}}all_csets{{optional_suffix}}(codeset_id);
+CREATE INDEX ac_idx1{{optional_index_suffix}} ON {{schema}}all_csets{{optional_suffix}}(codeset_id);
```

Comment:
Changed method of re-creating indexes. Now adds a random hash suffix rather than recreating at the end.